src/js/components/Card.js

```diff
@@ -74,9 +75,36 @@ export default class Card extends Component {
     return videoLayer;
   }
 
+  _renderParagraph (contents, textSize, type) {
+    if (typeof contents === 'string') {
+      return (
+        <Paragraph
+          className={`${CLASS_ROOT}__${type}`}
+          size={textSize}
+          margin="none"
+        >
+          {contents}
+        </Paragraph>
+      );
+    } else if (Array.isArray(contents)) {
```

Comment:
Oops forgot to change the propTypes on these. We ran into this problem on estories, where we should handle multiple paragraphs (see screenshot below). The new designs I've seen doesn't display multiple paragraphs, should I remove this feature? 

![image](https://cloud.githubusercontent.com/assets/3210082/18299731/e92167d2-745e-11e6-8262-f5e185a4758c.png)